ebuild-writing/misc-files/metadata/text.xml

```diff
@@ -271,8 +280,8 @@ There are also some attributes that can be used with these tags:
 <tr>
   <ti>restrict</ti>
   <ti>
-    <c>&lt;maintainer&gt;</c>,
-    <c>&lt;longdescription&gt;</c>, <c>&lt;flag&gt;</c>
+    <c>&lt;longdescription&gt;</c>, <c>&lt;maintainer&gt;</c>,
```

Comment:
I don't get the order here ;-).

Reply:
It's supposed to be the same ordering as in https://www.gentoo.org/glep/glep-0068.html#top-level-structure.